# Messaggi del classificatore

Header header

uint8 ARM_SYNCED = 1
uint8 ARM_UNSYNCED = 2
uint8 type

uint8 ARM_RIGHT = 1
uint8 ARM_LEFT = 2
uint8 arm

uint8 TOWARD_WRIST = 1
uint8 TOWARD_ELBOW = 2
uint8 x_direction

# For both ARM and X_DIRECTION
uint8 UNKNOWN = 255